contracts/core/Controller.sol

```diff
@@ -371,8 +373,6 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
      * @param _partialPauser new partialPauser address
      */
     function setPartialPauser(address _partialPauser) external onlyOwner {
-        require(_partialPauser != address(0), "C11");
```

Comment:
can we keep `require(_partialPauser != address(0), "C11");`

Reply:
same as above